working/augmentation-libraries/feature-specification.md

```diff
@@ -459,7 +398,7 @@ We say that *I* is the declaration which is *augmented by* *D*.
 *In other words, take all of the declarations with the same name in some
 augmentation context, order them according to the "after" relation, and each
 augments the result of all the prior augmentations applied to the original
-declaration . The first one must not be marked `augment` and all the subsequent
+declaration. The first one must not be marked `augment` and all the subsequent
```

Comment:
(Still think we should make `augment` optional, just like `override`. You can use it to document intent, but you don't have to. It'll try to augment either way, and fail in the same cases, but you also get an error if you write `augment` and it doesn't augment anything.)

Reply:
Filed an issue to discuss: https://github.com/dart-lang/language/issues/4377